### DISCUSSION TITLE
Contribute to elementary applications from other distros?

### DISCUSSION THREAD

**Discussion by ghost** [elementary/triage#324] [Q&A] [answered]
Hi!

I want to make a fix for one of the applications of elementary, Code, but I currently do not have my elementaryOS device. I only have a Windows 11 device with Ubuntu (WSL). Am I able to run Code or other applications from within bare Ubuntu?

**Comment by lenemter** [comment, marked as answer]
Yes, I think that's possible. Code has their dependencies available in Ubuntu repositories, so you can follow the instructions in Readme to compile it.

**Reply by RMcNeely** [reply]
If you clone the repo you should be able to install everything locally and develop on it. You might need to add the `elementary-sdk` package (though I'm not sure if that's in Ubuntu's repos) in order to get some of the development dependencies.
